plugins/dbviewer/frontend/public/javascripts/countly.models.js

```diff
@@ -6,20 +6,29 @@
 		_document = {};
 
     //Public Methods
-    countlyDBviewer.initialize = function () {
+    countlyDBviewer.initialize = function (app_id) {
+    	var data = { api_key: countlyGlobal['member'].api_key };
+		// is app_id provided?
+		if (app_id !== "" && app_id == "all") delete data.app_id;
+		else data.app_id = app_id;
+		// is there stored app_id?
+		if ((!app_id && store.get('selected_app')) && store.get('selected_app') !== "all") {
+			data.app_id = store.get('selected_app');
```

Comment:
selected_app storage key is too generic, as storage is shared, it might clash with others